src/modules/fancyzones/FancyZonesLib/WindowMoveHandler.cpp

```diff
@@ -101,12 +101,12 @@ void WindowMoveHandler::MoveSizeStart(HWND window, HMONITOR monitor, POINT const
         {
             for (auto [keyMonitor, workArea] : workAreaMap)
             {
-                // Skip calling ShowZoneWindow for iter->second (m_draggedWindowWorkArea) since it
+                // Skip calling ShowZonesDrawing for iter->second (m_draggedWindowWorkArea) since it
                 // was already called in MoveSizeEnter
                 const bool moveSizeEnterCalled = workArea == m_draggedWindowWorkArea;
                 if (workArea && !moveSizeEnterCalled)
                 {
-                    workArea->ShowZoneWindow();
+                    workArea->ShowZonesDrawing();
```

Comment:
In this case, would be better just _ShowZones_.

Reply:
I think it is clearer to use `ShowZonesDrawing` since it is more consistent (invoke the show method of `ZonesDrawing`).

Note that it makes sense as long as you think of `Drawing` in the sense of "sketch" and not the act of drawing. Another option would be to rename `ZoneWindow` to `ZonesOverlay` in order to emphasize it.

What do you think?

Reply:
`ZonesOverlay` is the best one IMO.

Reply:
Renamed to `ZonesOverlay`.